engine/classes/Elgg/Database/EntityTable.php

```diff
@@ -1221,7 +1196,13 @@ public function getEntityAttributeWhereSql(array $options = []) {
 		}
 
 		// @todo the types should be defined somewhere (as constant on \ElggEntity?)
-		if (!in_array($type, ['group', 'object', 'site', 'user'])) {
+		if (!in_array($type, [
```

Comment:
I think we should remove this from config and make it a static var somewhere: ``Application::$valid_types``